Tests/KeyStoreTests.swift

```diff
@@ -65,15 +65,15 @@ class KeyStoreTests: XCTestCase {
         let keyStore = try KeyStore(keyDirectory: keyDirectory)
         let wallet = keyStore.keyWallet!
         try keyStore.update(wallet: wallet, password: "testpassword", newPassword: "password")
-        let account = try wallet.getAccount(password: "password", coin: .ethereum)
+        let account = try wallet.getAccount(password: "password", coin: 60)
 
         XCTAssertNoThrow(try account.sign(hash: Data(repeating: 0, count: 32), password: "password"))
     }
 
     func testSigningMultiple() throws {
         let keyStore = try KeyStore(keyDirectory: keyDirectory)
         let wallet = keyStore.keyWallet!
-        let account = try wallet.getAccount(password: "testpassword", coin: .ethereum)
+        let account = try wallet.getAccount(password: "testpassword", coin: 60)
```

Comment:
can you use `Ethereum().coinType`  ? and etc.. otherwise it's hard to tell what 60 means here